scripts/cloud.py

```diff
@@ -159,7 +158,7 @@ def is_logged_in(aws_env: AwsEnv, use_aws_profiles: bool) -> bool:
         aws_env_as_profile = aws_env if use_aws_profiles else None
 
         sts = get_sts_client(aws_env_as_profile)
-        sts.get_caller_identity()
+        sts.get_caller_identity()  # pyright: ignore[reportAttributeAccessIssue]
```

Comment:
thought: There is a fix for these type hints if we want. It requires adding this package as a dependency `mypy-boto3-sts` and using the `STSClient` stub as a return type for the get client function. 

```python
from mypy_boto3_sts.client import STSClient

def get_sts_client(
    aws_env: Optional[AwsEnv],
) -> STSClient:
```